admin/config.yml

```diff
@@ -17,7 +17,7 @@ backend:
 #
 # backend:
 #   name: git-gateway
-#   branch: release-netlify-cms
+#   branch: LG-5431-javascript-contact-us
```

Comment:
does the branch name affect anything here?

Reply:
it does not. This gets changed when running locally, therefore there is no consequence to this line being changed once commented out.